### PR TITLE
Document meaninglessness of test tfstate values

### PR DESCRIPTION
**Pull request by craigfurman**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | 
| ❓ Documentation  | no

## Description

This repo contains many tfstate files for testing purposes. Resource
names and other values in here are just opaque strings, and no running
code attempts to access such resources.

Documenting this might prevent bug bounty hunters from filing security
reports about things like unclaimed S3 bucket names.

### CODE REVIEW

**Review by Tomer-Avraham — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -76,3 +76,12 @@ Build with ❤️️ from 🇫🇷 🇬🇧 🇯🇵 🇬🇷 🇸🇪 🇺🇸
...
 Feel free to book a slot.
+
+---
+
+## Security notice
+
+All Terraform state files (`*.tfstate`) in this repository are for unit test
+purposes only. They do not reference resources in any particular cloud
+environment, and no running code attempts to access them. They are just opaque
+strings.
```
Comment (eliecharra) [inline README.md:87]: We also have a bunch of `.tf` files that are never applied but that can be used to recreate some testing context for local dev. I think we should we reference theses files here, not only the states.

I'm also wondering if we can extend the scope of this notice to the whole repo for both `.tfstate` and `.tf` files.
We do not plug any real cloud env on them and the only one that are applied are the ones for acceptance testing.
It still does not make sense to trigger security issues on theses files. BTW we may also want to have assumed vulnerable files here for testing purposes.

**Review by eliecharra — COMMENTED** [review]
I'm fine with that, but I think we can scope the whole repo instead of only test folders

File: README.md (hunk 1 above)
Comment (craigfurman) [reply README.md]: Thanks @eliecharra, good idea. I've removed the deeply-nested readme, and updated the notice in the top-level readme to apply to TF (not just state) and made the message broader and shorter. WDYT?

**Review by eliecharra — APPROVED** [review]
Thanks a lot for this quick fix 👍🏻

**Pull request merged**